packages/documentation/components/ComponentInfo.vue

```diff
@@ -253,7 +253,7 @@ export default defineComponent<{
 				return result
 			}),
 			showProps: ref(false),
-			stringifyDefault: (defaultValue: Function | unknown) =>
```

Comment:
todo: check configuration and investigate if it would be useful to extend the rules of `@typescript-eslint/ban-types`
to suit our mistakes
(e.g. using `Boolean` interface, and `String` interface on props defintions, and defining `object` types as `{}` (saw it once, might be gone now)

Reply:
https://www.typescriptlang.org/play?#code/PTAEBcE8AcFNQEoHsCu54F5QG8BQJQBIaAQ3AAsAuUAZ3ACcBLAOwHN8xDmSBbWAfmp0mbUAB9QzFABtpHIuRI0qtBi1ah5AX1zyocUAA1QWbKADaAa1iQhatgF1qyNPABkOUPVgATRt4BjcGdUdFAtcM15AKRmOlAAD2pjU3liMhUAcgBGAAZMgBo0xWVqTMK07j5qFGYfWAAzFl8igkIAfW8-QOCcNPSKMryKtsISrOyRzi5eWBq6xuafNJ0CHVx9eAAFWHoaWJM+merVEXZCElY5yRQeACNdzXXN0B292IB1RgoAaUYfGiHN77ZigDzYTq+fywILUACC9HoJEgAB5gbEAHzrGJxcCgBrSJDeZi9dHML6-f40VLHa6ZAlE2AkioXK7UABMuQKmg6XWhsIseEItLKARI9CQ0hZrOueSKhC0Dlw2Ni8UM1CQdwAVjC8VgpLJQEA